firmware/application/hw/encoder.cpp

```diff
@@ -50,5 +102,10 @@ int_fast8_t Encoder::update(
     state <<= 1;
     state |= phase_1;
 
-    return transition_map[state & 0xf];
+    // get dial sensitivity setting from pmem if valid
+    uint8_t sensitivity = portapack::persistent_memory::config_encoder_dial_sensitivity();
```

Comment:
Is this fast to retrieve or should this be cached in the portapack namespace in memory? (I don't know)